src/tmpfiles/tmpfiles.c

```diff
@@ -2261,11 +2261,12 @@ static int process_item(Item *i, OperationMask operation) {
 
         i->done |= operation;
 
-        r = chase_symlinks(i->path, NULL, CHASE_NO_AUTOFS, NULL);
+        r = chase_symlinks(i->path, NULL, CHASE_NO_AUTOFS|CHASE_WARN, NULL);
         if (r == -EREMOTE) {
-                log_debug_errno(r, "Item '%s' is below autofs, skipping.", i->path);
+                log_warning_errno(r, "Skipping.");
```

Comment:
maybe log_notice, and I think it would be smart to say here what is skipped, i.e. "Skipping %s", i->path)